docs/glossary.md

```diff
@@ -1,6 +1,6 @@
 This glossary defines a number of terms commonly used across the documentation. For the definitions of framework components consult the API docs.
 
-* **AEA (Autonomous Economic Agent)**: An AEA is "an intelligent agent acting on an owner's behalf, with limited or no interference, and whose goal is to generate economic value to its owner". AEAs are a special type of agent. [<a href="../index">more</a>]
+* **AEA (Autonomous Economic Agent)**: An AEA is "an intelligent agent acting on an owner's behalf, with limited or no interference, and whose goal is to generate economic value to its owner". AEAs are a special type of agent. [<a href="../agent-oriented-development">more</a>]
```

Comment:
I think the original reference was correct as we define AEA on the index page.

Reply:
turns out you cannot reference index.md files like that, and need to use [<a href="../">more</a>] to reference there